src/main/java/de/florianmichael/viafabricplus/util/ChatUtil.java

```diff
@@ -23,16 +23,21 @@
 
 public class ChatUtil {
     public final static String PREFIX = Formatting.WHITE + "[" + Formatting.GOLD + "ViaFabricPlus" + Formatting.WHITE + "]";
```

Comment:
Can you generate the PREFIX field using the PREFIX_TEXT field? (maybe we can remove it completely by just using getString()?)

Reply:
1. Technically yeah, but not easily
2. `.getString()` strips formatting, so no

Reply:
fair enough